api/internal/handler/data_loader/route_export.go

```diff
@@ -233,7 +235,7 @@ func (h *Handler) RouteToOpenAPI3(c droplet.Context, routes []*entity.Route) (*o
 			return nil, err
 		}
 
-		if plugins != nil {
+		if len(plugins) > 0 {
```

Comment:
how about add `plugins != nil &&`

Reply:
It was written in the past(commit:https://github.com/apache/apisix-dashboard/pull/1673/commits/6dd85785cd752539b0ec50652e68e1f34ef1cf56), but go-link will report error, so it was modified according to go-link.

Reply:
ok..